utilities/src/main/java/io/syndesis/qe/utils/DbUtils.java

```diff
@@ -75,4 +103,59 @@ public void deleteRecordsInTable(String tableName) {
 			log.error("Error: " + ex);
 		}
 	}
+
+	public Integer insertInDB(String tableName, List<List<String>> dataTable) {
```

Comment:
I wonder if the same result could be achieve by `executeSqlOnSampleDb()` and processing the result outside DbUtil class. This class should stay pretty genetic and the testing logic executed in verifying steps.

cc @tplevko wdyt?